pkg/chunked/storage_linux.go

```diff
@@ -229,24 +231,33 @@ func makeZstdChunkedDiffer(store storage.Store, blobSize int64, tocDigest digest
 	if err != nil {
 		return nil, fmt.Errorf("read zstd:chunked manifest: %w", err)
 	}
+	var uncompressedTarSize int64 = -1
+	if tarSplit != nil {
+		uncompressedTarSize, err = tarSizeFromTarSplit(tarSplit)
+		if err != nil {
+			return nil, fmt.Errorf("computing size from tar-split")
```

Comment:
nit: `errors.New`, or `fmt.Errorf("computing site from tar-split: %w", err)`, or just `err`.

Reply:
Thanks! Fixed in https://github.com/containers/storage/pull/2136 .